src/app/screens/SignUp/index.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { useRef } from 'react';
+import i18next from 'i18next';
+import { useForm } from 'react-hook-form';
+
+import User from '../../../typings/user';
+import logo from '../../assets/logo-wolox.png';
+import Button from '../../components/Button';
+import Input from '../../components/Input';
+
+import styles from './styles.module.scss';
+
+function SignUp() {
+  const { register, errors, handleSubmit, watch } = useForm<User>({ mode: 'all' });
+
+  const password = useRef('');
+  password.current = watch('password', '');
+
+  const requireMessage = i18next.t('SignUp:required_field');
+  const passwordDontMatch = i18next.t('SignUp:passwords_dont_match');
+
+  const onSubmit = handleSubmit((data) => {
+    console.log({ user: { ...data, locale: 'en' } });
+  });
+
+  return (
+    <div className={styles.container}>
+      <div className="row center middle">
+        <div className={styles.containerSignUp}>
```

Comment:
Are you sure that you need all these divs elements? avoid using the divs patterns

Reply:
You're right, I think I added a totally unnecessary div